common/schemas/keyman-touch-layout/README.md

```diff
@@ -399,6 +399,13 @@ and `width`.
 : Number. Defines the width of the key. Not currently supported for longpress
   keys. See `key` object for details.
 
+* `default`
+
+: Boolean. Defaults to `false`. If true, this will be the default key when the
+  user longpresses the base key without moving their finger. Only one subkey
+  should be default; if multiple subkeys have the default field set, then
+  behaviour is undefined.
```

Comment:
It might be nice to have the "cancel" behavior you wrote in #8519 documented somewhere

> * Given that cancel is _not_ the preferred default action for a longpress, the user can cancel the longpress default by sliding outside the 'acceptance' area -- e.g. south of the key cap, which would then unhighlight the default and indicate visually that no output would occur when they release their finger.